Hindi/hindi.md

```diff
@@ -0,0 +1 @@
+?? ??? ????? ????? ??? ??????? ???? ??? ??? ??? ????? ?? ???? ???? ??? ????? ????? ?? ????? ?????? ???? ?? ??? ?? ??? ??? ????? ??? ?? ???? ??, ?? ?? ????? ????? ??? ??????? ???? ???? ??, ??? ????????? ?? ???? ??? ???? ??? ?? ?? ??? ?? ???? ???
```

Comment:
Hi Caroline ,
I'll fix that type it in english and resubmit it on friday.
I have a paper submission so am tied up.
Hope that is okay.

Regards,
Nikhil

On Tue, Jan 20, 2015 at 7:40 PM, Caroline Buckey <notifications@github.com>
wrote:

> In Hindi/hindi.md
> <https://github.com/udacity/create-your-own-adventure/pull/568#discussion_r23270196>
> :
>
> > @@ -0,0 +1 @@
> > +?? ??? ????? ????? ??? ??????? ???? ??? ??? ??? ????? ?? ???? ???? ??? ????? ????? ?? ????? ?????? ???? ?? ??? ?? ??? ??? ????? ??? ?? ???? ??, ?? ?? ????? ????? ??? ??????? ???? ???? ??, ??? ????????? ?? ???? ??? ???? ??? ?? ?? ??? ?? ???? ???
>
> I haven't used hindi fonts, so I don't know whether it's possible to get
> GitHub to display them correctly. Sorry I can't help with this problem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/568/files#r23270196>
> .
>